pubtools/_quay/tag_docker.py

```diff
@@ -579,7 +595,17 @@ def merge_manifest_lists_sign_images(self, push_item, tag, add_archs, signature_
                     )
                     claim_messages.append(message)
 
-        signature_handler.sign_claim_messages(claim_messages, True, True)
+        # claim messages ensure that signatures of non-modified archs won't get removed
+        self.signature_remover.remove_tag_signatures(
+            reference=dest_image,
+            pyxis_server=self.target_settings["pyxis_server"],
+            pyxis_krb_principal=self.target_settings["iib_krb_principal"],
+            pyxis_krb_ktfile=self.target_settings.get("iib_krb_ktfile", None),
+            exclude_by_claims=claim_messages,
+        )
+
+        filtered_claim_messages = signature_handler.filter_claim_messages(claim_messages)
+        signature_handler.sign_claim_messages(filtered_claim_messages, True, True)
```

Comment:
The second 'True' is to enable filter_claim_messages.

Reply:
:man_facepalming: :man_facepalming: :man_facepalming: fixed